CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 ### Added
+- Ability to generate an OpenAPI spec - @mainx07, @hudayou, @ruslantalpa, @begriffs
```

Comment:
Thanks, dude, I appreciate that, but I really see that as a community effort for all the time all the people spent on it. 👍